style.css

```diff
@@ -91,4 +98,53 @@
     padding: 40px 100px 50px 100px;   
     outline: none;
     cursor: pointer;
+}
+
+@media screen and (max-width: 850px) {
+    .main {
+        flex-direction: column;    
+    }
+  }
+@media screen and (max-width: 450px){
+    
+    .intestazione{
+        width: 260px;
+        height: 260px; 
+    }
+
+    .login{
+        width: 260px;
+        height: 410px;
+    }
+    
+    .intestazione p{
+        display: none;
+    }
+
+}
+/*A questa dimensione i bottoni non si dimensionano più in funzione del div */
+
+@media screen and (max-width: 350px){
+    
+    .intestazione{
+        width: 160px;
+        height: 160px; 
+    }
+
+    .login{
+        width: 160px;
+        height: 310px;
+    }
+    .login #btn1{
+        flex-basis: 10%;
+        flex-shrink: 1;
+    }
+    .login #btn2{
```

Comment:
se i due bottoni hanno lo stesso stile 
.login #btn1 e .login #btn2
è meglio utilizzare una class .login .button {
 flex-basis: 10%;
 flex-shrink: 1;
}

cosi se devi cambiare lo stile a più bottoni perchè un cliente cambia idea tu lo modifichi in un punto solo e la modifica si propaga